README.md

```diff
@@ -110,8 +109,11 @@ workflow: This package contains any subtask the service might have. Each sub-pac
 
 Any test files can be found under test/ folder. The directory under this folder should exactly match that in src/ with each test file in those packages testing the corresponding source file. Currently, this is not setup and is a work in progress and will basically emulate the above once all the tests have been written.
```

Comment:
can rephrase to indicate to add tests you can add to the corresponding package of src, or have some other section "Adding Tests"